packages/css/src/tile/tile.css

```diff
@@ -13,20 +13,28 @@
   gap: 1.5em;
   text-align: start;
   width: 440px;
+  min-width: 440px;
```

Comment:
Dette er for å hindre at tilen i "button"-type blir flexet ned til mindre enn 440px.